drivers/hmis/spec/factories/hmis/hud/project_configs.rb

```diff
@@ -9,4 +9,10 @@
     created_at { Date.parse('2019-01-01') }
     updated_at { Date.parse('2019-01-01') }
   end
+
+  factory :hmis_project_auto_exit_config, class: 'Hmis::ProjectAutoExitConfig' do
+    created_at { Date.parse('2019-01-01') }
+    updated_at { Date.parse('2019-01-01') }
```

Comment:
Can you switch all these to `Time.current`? The timestamp fields should be DateTimes not Dates

Reply:
And as an aside, we always make sure to use `.current` (NOT `.now` or `.today`) in tests (and everywhere, really) because they can cause timezone issues: https://discuss.rubyonrails.org/t/time-now-vs-time-current-vs-datetime-now/75183

Reply:
Cool, thank you!